src/content/docs/logs/forward-logs/enable-log-management-new-relic.mdx

```diff
@@ -75,7 +75,18 @@ Here are some details on how to accomplish specific use cases:
         Collect logs from apps running in Docker containers, either standalone or when using Kubernetes
       </td>
       <td>
-        Use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) or our [infrastructure monitoring agent with Docker](/docs/infrastructure/install-infrastructure-agent/linux-installation/docker-container-infrastructure-monitoring/#requirements).
+        For Kubernetes environments, use our [Kubernetes plugin](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/).
+        
+        For Docker environments, we offer a few options:
+        * Use our [APM logs in context](/docs/apm/new-relic-apm/getting-started/get-started-logs-context)
```

Comment:
@lchapman4 For the logs in context option, do they need to do anything additional for Docker? Does that mean that all Docker customers will automatically have their host logs collected _because_ they're using APM agents?

Reply:
For the apm logs-in-context option nothing else is needed on their part. That said, if they use the apm method they will not collect host-level logs. That's a natural limitation of docker. To collect host level logs they would need to use our infra agent installed on the host.

Reply:
Hey @ally-sassman sorry, I lost track of this. Is anything else needed for this PR?